firmware/application/apps/ui_freqman.cpp

```diff
@@ -309,31 +337,44 @@ FrequencyManagerView::FrequencyManagerView(
     };
 
     button_add_category.on_select = [this]() {
+        // this one will only directly add to /FREQMAN
         on_add_category();
     };
 
-    button_del_category.on_select = [this]() {
-        on_del_category();
+    button_del_category.on_select = [this, &nav]() {
+        if (!forbid_delete_system_item_helper(nav)) {
```

Comment:
Deleting a category means deleting a FreqMan file.  For this cate, the "do it with the Notepad app" text isn't entirely accurate.  I think it should probably say "File Manager" in the "del_category" case.

Reply:
Thanks!
Actually it's because previously you asked me to change to this one.

I will use a more general expressing because it's not worth to save two strings in flash.